CMakeLists.txt

```diff
@@ -137,6 +138,9 @@ if (BENCHMARK_BUILD_32_BITS)
   add_required_cxx_compiler_flag(-m32)
 endif()
 
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

Comment:
Probably also want `CMAKE_CXX_EXTENSIONS OFF`

Reply:
yeah.. i'm also eyeing up the check where we:

```
set(EXTRA_CXX_FLAGS "")
if (WIN32 AND "${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang")
  # Clang on Windows fails to compile the regex feature check under C++11
  set(EXTRA_CXX_FLAGS "-DCMAKE_CXX_STANDARD=14")
endif()
```

and wondering how that will end up interacting with this change.

Reply:
yes FWIW we need this check for clang on windows.